src/gui/src/lib/external-info.ts

```diff
@@ -137,6 +156,19 @@ export const retrieveAvatar = async (
   return `https://gravatar.com/avatar/${hash}?d=404`
 }
```

Comment:
Let's refactor `retrieveGitHubAPIUrl` to share the same logic to sanitize the github url from src/gui/src/utils/get-repo-url.ts